bin/reportdaily.py

```diff
@@ -56,6 +64,142 @@ class MissingSubCommand(ValueError):
 log.addHandler(logging.NullHandler())
 
 
+def cmd_init(args):
+    """Creates an initial config file for a user config file"""
+
+    # create config by hand
+    check_if_config_exists(args)
+
+    log.debug("INIT selected %s", args)
+    print("Init selected", args)
```

Comment:
You don't need the `print` as you already have a `log.debug`. Furthermore, I'd recommend to move the `log.debug` line as a first command. If there is an error before, you won't see no debug message.